database/seed.js

```diff
@@ -12,58 +11,60 @@ const random = (min, max, floor) => {
 };
 
 const newBoolean = () => {
-  let num = random(0, 100, true);
+  const num = random(0, 2, true);
   if (num % 2 === 0) {
     return true;
   }
   return false;
 };
 
 const randRating = () => {
-  let chance = random(0, 100, true);
-  if (chance > 20) {//arbitrarily set chance of reviews below 4 to 20%
-    let num = random(0, 1, false);
+  const chance = random(0, 100, true);
+  if (chance > 20) { // arbitrarily set chance of reviews below 4 to 20%
+    const num = random(0, 1, false);
```

Comment:
May want to change random function to make sure the decimal doesn't go beyond 2 characters.